src/pkg/reg/adapter/aliacr/openapi_test.go

```diff
@@ -0,0 +1,262 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aliacr
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/goharbor/harbor/src/pkg/reg/model"
+)
+
+var acrAdapter *adapter
+var acreeAdapter *adapter
+
+func init() {
+	accessKey := os.Getenv("ALIYUN_ACCESS_KEY")
```

Comment:
Looking at the code, we need to use the real Alibaba Cloud AK SK to run UTs. Can it be mock?

Reply:
> Looking at the code, we need to use the real Alibaba Cloud AK SK to run UTs. Can it be mock?

@kofj Mock openapi seems not to have achieved the purpose of testing it. Do you have any suggestions for this part? My first thought was to provide an AK/SK to assist testing.